.github/CODEOWNERS

```diff
@@ -71,19 +71,17 @@ BUILD @christophermaier
 /src/rust/grapl-service/ @inickles-grapl
 /src/rust/grapl-utils/ @inickles-grapl
 /src/rust/grapl-web-ui/ @inickles-grapl
+/src/rust/Makefile @christophermaier
 /src/rust/model-plugin-deployer/ @wimax-grapl
+/src/rust/node-identifier/ @grapl-security/wg-data-infra
```

Comment:
FYI this team doesn't appear to exist according to github. Can you double-check the name here?